sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/CosmosAuthorizationTokenResolverTest.java

```diff
@@ -95,270 +91,308 @@ public CosmosAuthorizationTokenResolverTest(AsyncDocumentClient.Builder clientBu
 
     @BeforeClass(groups = { "simple" }, timeOut = SETUP_TIMEOUT)
```

Comment:
fyi- group is "simple" and not "emulator", so it wont run on default emulator ci and full ci have to run

Reply:
I'll change the test grouping to emulator, since we don't seem to have any coverage for resource token usage in the simple CI.